src/libcommon/utility/types.h

```diff
@@ -100,7 +100,16 @@ static constexpr std::string_view clientCrashFileName(
 static constexpr std::string_view clientKillFileName(
         "kdrive_client.kill"); // Name of the file generated by the crash handler when a kill of the client occurs
 
-enum class AppType { Server, Client };
+static constexpr std::string_view serverCrashEventFileName(
```

Comment:
Why not defining these new strings and types where there are used?

Reply:
This class is used in quite a few places (appclient, appserver, mainclient, mainserver, sentryhandler, utility). It was a duplicate, I just deleted the duplicate.

Reply:
Good to know, thanks! What about a namespace to keep the strings together?

Reply:
Done